src/exports.h

```diff
@@ -1,35 +1,26 @@
 #ifndef _SCHAUFEL_EXPORTS_H_
 #define _SCHAUFEL_EXPORTS_H_
 
-#include <consumer.h>
-#include <utils/logger.h>
-#include <producer.h>
-#include <stdlib.h>
-#include <libpq-fe.h>
-#include <pthread.h>
-#include <json-c/json.h>
-#include <queue.h>
-#include <sys/prctl.h>
-#include <endian.h>
-
-typedef struct Producer *Producer;
+#include <libconfig.h>
+
```

Comment:
We'll need to add
`#include <endian.h>`
here. It's not a standard include (glibc implicitly includes when _BSD_SOURCE is set for no reason).